sql/sql_expression.cc

```diff
@@ -3832,7 +3832,16 @@ SelectStatement::parse(ML::Parse_Context& context, bool acceptUtf8)
     skip_whitespace(context);
 
     //cerr << jsonEncode(statement) << endl;
-    
+
+    if (statement.from->getType() == "null") {
+        for (const auto & c: statement.select.clauses) {
+            if (typeid(*c.get()) == typeid(WildcardExpression)) {
+                throw ML::Exception(
+                    "Wildcard usage requires a FROM statement");
```

Comment:
It doesn't go in the parser.  It should go in the query executor.  The parser deals with syntax, not semantics.